tests/test_crash_similarity.py

```diff
@@ -86,6 +86,38 @@ def test_wmdistance_zero_distance(self):
 
         self.assertEqual(float('inf'), distance)
 
+    def test_wmdistance_euclidean_non_zero_distance(self):
```

Comment:
Nit: It would be good to refactor the tests to avoid duplicated code, but it's not a big deal.